source/site_ops/how-tos/create_reusable_lti_configuration.rst

```diff
@@ -0,0 +1,42 @@
+.. _Create a Reusable LTI 1.3 Configuration:
+
+
+Create a Reusable LTI 1.3 Configuration
+######################################################
+
+.. tags:: site operator, how-to
+
+
+Overview
+********
+
+Reusable LTI configurations are created once by an administrator in Django Admin (LTI Store) and identified by a filter key.  
+In Studio, you can point an LTI Consumer component to that reusable configuration, avoiding repeated copy/paste of credentials.
+
+See :ref:`Set up an LTI Consumer with Reusable LTI Configuration` for
+instructions on using the configuration in Studio once it has been set up.
+
+Create Reusable Configuration in Django Admin
+*********************************************
+
+* Create a reusable LTI 1.3 configuration in Django Admin.
```

Comment:
How do you do this?

Reply:
Yes, that's helpful. That's the doc I'm moving to a site-ops how-to. It's not appropriate for Educators.